### ISSUE TITLE
Add switch for auto close

### ISSUE THREAD

**Issue by JimDubs** [A-Coom/coomer.party-scraper#12]
Running the scripts under a master bat file with calls to each scrape bat file, they don’t progress because each ask for an enter to close.

perhaps add a -auto switch to auto complete once done?

**Comment by A-Coom** [comment]
Good idea! I don't see the need to add another parameter. An easy change would be to only ask for an enter to exit if the user is prompted for the parameters after execution. In this way, your problem would be solved and users who just double-click the EXE will know the program didn't crash.

The holidays prevent me from adding this until next week, but will approve a suitable pull request to the dev branch.

**Comment by lee5002** [comment]
I just created a PR for fixing this. Please feel free to merge it: https://github.com/A-Coom/coomer.party-scraper/pull/15
